src/lib/style.store.ts

```diff
@@ -0,0 +1,26 @@
+import { readable, type Readable } from "svelte/store";
+
+export interface Logo {
+  dark?: string;
+  height?: number;
+  light: string;
+  width?: number;
+}
+export interface Style {
+  checksAndRadios?: 'animated' | 'standard';
+  labels?: 'floating' | 'stacked';
+  logo?: Logo;
+  sections?: {
+    header?: boolean;
+  };
+  stage?: {
+    icon: boolean;
+  };
+}
+
+export let style: Readable<Style>;
+
+// TODO: Implement Zod for better usability
+export function initialize(customStyle: Style) {
+  style = readable(customStyle);
+}
```

Comment:
This file will be the store for all style controls of the Widget. This allows us to not have to pass a ton of props around the app.